templates/binding.gyp.hbs

```diff
@@ -45,12 +44,6 @@ Required context:
             "outputs": ["../<(static_lib)"],
```

Comment:
The same case as the previous one. It avoid to create unnecessary `target` directory in `..`

```
"libraries": ["<(PRODUCT_DIR)/../../<(static_lib)"],
```